go/bind/keybase.go

```diff
@@ -4,6 +4,7 @@
 package keybase
 
 import (
+	"context"
```

Comment:
Oh that's right! Hmm why was I under the impression it had to be the same package ...